.travis.yml

```diff
@@ -1,99 +1,25 @@
-language: php
-
-
-cache:
-    directories:
-        - vendor
+sudo: required
 
-matrix:
-  include:
-  - php: "7.2"
-  - php: "7.3"
-    env: ALLOW_FAILURE='true'
-  allow_failures:
-  - php: "7.3"
-    env: ALLOW_FAILURE='true'
+# any language should do it
+language: php
 
-addons:
-    firefox: "44.0"
```

Comment:
What browser/version does this test with now?

Reply:
firefox 47.0.1
I will update the version later, not in this PR.